### PR TITLE
Community Profile Daniel Kiptoon

### LINKED ISSUE #5026: [Community] Member Profile: Daniel Kiptoon
**Current Behavior**

@KiptoonKipkurui has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @KiptoonKipkurui as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @KiptoonKipkurui 
- Twitter: 
- LinkedIn: [daniel-kipkurui-kiptoon-56b8a1177](https://www.linkedin.com/in/daniel-kipkurui-kiptoon-56b8a1177/)
- Meshery Cloud: [a1837b00-c5b8-4a91-86d6-dec9527d70be](https://meshery.layer5.io/user/a1837b00-c5b8-4a91-86d6-dec9527d70be)
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by KiptoonKipkurui**
**Description**
This is a community profile for Daniel Kiptoon
This PR fixes #5026 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by vishalvivekm — APPROVED** [review]
## Yay!🎈@KiptoonKipkurui 😎

**Comment by l5io** [comment]
🚀 Preview for commit 8f379e1e85470054b3815e3ed926a6449dae3347 at: https://6536c17563684c744e904256--layer5.netlify.app

File: src/collections/members/daniel-kiptoon/index.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+twitter: danchi101
+linkedin: daniel-kipkurui-kiptoon-56b8a1177
+layer5: a1837b00-c5b8-4a91-86d6-dec9527d70be
+location: Nairobi, Kenya
+bio: "Daniel Kiptoon is a seasoned software engineer with a fervent passion for creating scalable systems, mastering the intricacies of distributed computing (Kubernetes and Docker), and promoting DevSecOps principles. He aspires to improve the technology landscape with innovative solutions and to lead the conversations on the next generation cloud-native solutions."
+badges: 
+   - meshery-operator
+blurb: Distributed and Parallel
+status: Active
+maintainer: no
```
Comment (vishalvivekm) [inline src/collections/members/daniel-kiptoon/index.mdx:15]:
```suggestion
```

Comment (vishalvivekm) [reply src/collections/members/daniel-kiptoon/index.mdx:15]: > @vishalvivekm I wonder though why its made me a MeshMate even though I haven't explicitly configured it in the template

@KiptoonKipkurui removing this line fixes it. Seems to me a potential bug though.

**Review by vishalvivekm — CHANGES_REQUESTED** [review]

File: src/collections/members/daniel-kiptoon/index.mdx (hunk 1 above)
Comment (vishalvivekm) [reply src/collections/members/daniel-kiptoon/index.mdx:15]: > @vishalvivekm I wonder though why its made me a MeshMate even though I haven't explicitly configured it in the template

@KiptoonKipkurui removing this line fixes it. Seems to me a potential bug though.

**Comment by l5io** [comment]
🚀 Preview for commit fede9e42c98804a761600b5b77162aca561e69d8 at: https://6536d7cec948211f5953e1b3--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]
Awesome. 🙂

File: src/collections/members/daniel-kiptoon/index.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+twitter: danchi101
+linkedin: daniel-kipkurui-kiptoon-56b8a1177
+layer5: a1837b00-c5b8-4a91-86d6-dec9527d70be
+location: Nairobi, Kenya
+bio: "Daniel Kiptoon is a seasoned software engineer with a fervent passion for creating scalable systems, mastering the intricacies of distributed computing (Kubernetes and Docker), and promoting DevSecOps principles. He aspires to improve the technology landscape with innovative solutions and to lead the conversations on the next generation cloud-native solutions."
+badges: 
+   - meshery-operator
+blurb: Distributed and Parallel
+status: Active
+meshmate: no
```
Comment (vishalvivekm) [inline src/collections/members/daniel-kiptoon/index.mdx:15]:
```suggestion
```

File: src/collections/members/daniel-kiptoon/index.mdx (hunk 1 above)
Comment (vishalvivekm) [reply src/collections/members/daniel-kiptoon/index.mdx]: My bad. I meant `meshmate: no` line @KiptoonKipkurui

**Pull request merged**